From e2b37e23a45709f96e101238bbd622457a23af3f Mon Sep 17 00:00:00 2001
From: Jakub Ruzicka <jruzicka@redhat.com>
Date: Mon, 8 Jul 2013 15:22:05 +0200
Subject: [PATCH] Remove runtime dependency on python-pbr.

---
 keystoneclient/__init__.py | 5 +----
 setup.py                   | 1 -
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/keystoneclient/__init__.py b/keystoneclient/__init__.py
index ab4f7a1..f735877 100644
--- a/keystoneclient/__init__.py
+++ b/keystoneclient/__init__.py
@@ -13,10 +13,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import pbr.version
-
-
-__version__ = pbr.version.VersionInfo('python-keystoneclient').version_string()
+__version__ = "REDHATKEYSTONECLIENTVERSION"
 
 __all__ = [
     # Modules
diff --git a/setup.py b/setup.py
index 70c2b3f..afab729 100644
--- a/setup.py
+++ b/setup.py
@@ -18,5 +18,4 @@
 import setuptools
 
 setuptools.setup(
-    setup_requires=['pbr'],
     pbr=True)
